## 75% of end-end tests have passed
```
Total executed tests: 8
Failed tests: 1
Expected to fail tests: 1
Passed tests: 6
```
```
IDs of failed tests: 703133 
```
```
IDs of expected to fail tests: 702843 
```
<h3>Failed tests (1)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=fix/tx-confirmation-crash&PR_ID=21695&APK_NAME=21695.apk&TR_CASE_IDS=703133">Rerun failed tests</a></li><br/><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3736260">test_restore_multiaccount_with_waku_backup_remove_profile_switch</a>, id: 703133 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: Button </div><div># STEP: Check that removed user is not shown in the list anymore</div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:240: in test_restore_multiaccount_with_waku_backup_remove_profile_switch
    self.errors.verify_no_errors()
base_test_case.py:192: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 zQ3...dWXh5 was not restored as a contact from waku backup!
E    zQ3...Vacac was not restored as a contact from waku backup!
E    admin_open was not restored from waku-backup!!
E    member_open was not restored from waku-backup!!
E    admin_closed was not restored from waku-backup!!
E    member_closed was not restored from waku-backup!!
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a7c437ee66294702b890ba8ffe129a5c?auth=db3f64e91183b009226fefa631ff5ba2#78">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Expected to fail tests (1)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3736259">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td>

```
Test is not run, e2e blocker  
```

<b>[[reason: [NOTRUN] Skipped due to waku issue on staging fleet]]</b><br/><br/></td></tr></tbody></table></details><h3>Passed tests (6)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3736262">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3736261">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3736258">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a04036e417f14f8eb1b97844ad735bec?auth=be4d252530eeb39d9434a0befdffb650#103">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/986733896adf4167a844b9ca4cb7dc37?auth=3431e7eb7389a75181a7fab5a257e0ee#98">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3736257">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a7c437ee66294702b890ba8ffe129a5c?auth=db3f64e91183b009226fefa631ff5ba2#46">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3736263">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3736264">test_wallet_balance_mainnet</a>, id: 740490 </b></td></tr><tr><td></td></tr></tbody></table></details>

--- a/src/status_im/contexts/wallet/send/transaction_confirmation/view.cljs
+++ b/src/status_im/contexts/wallet/send/transaction_confirmation/view.cljs
@@ -286,37 +286,33 @@
            :gradient-cover?          true
            :customization-color      (:color account)}
           [rn/view
-           [rn/pressable
-            {:on-press #(rf/dispatch [:navigate-to-within-stack
-                                      [:screen/wallet.transaction-details
-                                       :screen/wallet.transaction-confirmation]])}
-            [transaction-title
-             {:token-display-name token-symbol
-              :amount             amount
-              :account            account
-              :type               type
-              :recipient          recipient
-              :route              route
-              :to-network         bridge-to-network
-              :image-url          image-url
-              :transaction-type   transaction-type
-              :collectible?       collectible?}]
-            [user-summary
-             {:summary-type        :status-account
-              :accessibility-label :summary-from-label
-              :label               (i18n/label :t/from-capitalized)
-              :account-props       from-account-props
-              :theme               theme}]
-            [user-summary
-             {:summary-type        (if (= transaction-type :tx/bridge)
-                                     :status-account
-                                     :account)
-              :accessibility-label :summary-to-label
-              :label               (i18n/label :t/to-capitalized)
-              :account-props       (if (= transaction-type :tx/bridge)
-                                     from-account-props
-                                     user-props)
-              :recipient           recipient
-              :bridge-tx?          (= transaction-type :tx/bridge)
-              :account-to?         true
-              :theme               theme}]]]]]))))
+           [transaction-title
+            {:token-display-name token-symbol
+             :amount             amount
+             :account            account
+             :type               type
+             :recipient          recipient
+             :route              route
+             :to-network         bridge-to-network
+             :image-url          image-url
+             :transaction-type   transaction-type
+             :collectible?       collectible?}]
+           [user-summary
+            {:summary-type        :status-account
+             :accessibility-label :summary-from-label
+             :label               (i18n/label :t/from-capitalized)
+             :account-props       from-account-props
+             :theme               theme}]
+           [user-summary
+            {:summary-type        (if (= transaction-type :tx/bridge)
+                                    :status-account
+                                    :account)
+             :accessibility-label :summary-to-label
+             :label               (i18n/label :t/to-capitalized)
+             :account-props       (if (= transaction-type :tx/bridge)
+                                    from-account-props
+                                    user-props)
+             :recipient           recipient
+             :bridge-tx?          (= transaction-type :tx/bridge)
+             :account-to?         true
+             :theme               theme}]]]]))))

